CONTRIBUTING.md

```diff
@@ -10,4 +10,4 @@ Taichi is developed mainly in C++17 and Python3. Please check out the [Developer
 
 Issues marked with ["welcome contribution"](https://github.com/taichi-dev/taichi/issues?q=is%3Aopen+is%3Aissue+label%3A%22welcome+contribution%22) are great places for starters. You can quickly get an idea of the entire workflow and how to join the community.
 
-**RFC**: We use the `RFC` (Request for Comments) mechanism to discuss and organize some of the more advanced and self-contained features. These are the projects that we would like to work on but still lack a concrete design or implementation roadmap for because of their complexity. We document these requests and the threaded proposals in the hope that we could provide the community with a good enough context and draw upon insights from the potentially passionate minds. You can find all the ongoing RFCs [here](https://github.com/taichi-dev/taichi/issues?q=is%3Aissue+is%3Aopen+label%3ARFC+), and you are also welcome to file new RFCs with us!
+**RFC**: We use the `RFC` (Request for Comments) mechanism to discuss and organize some of the more advanced and self-contained features. These are the projects that we would like to work on but still lack a concrete design or implementation roadmap for their complexity. We document these requests and the threaded proposals in the hope that we could provide the community with a good enough context and draw upon insights from the potentially passionate minds. You can find all the ongoing RFCs [here](https://github.com/taichi-dev/taichi/issues?q=is%3Aissue+is%3Aopen+label%3ARFC+), and you are also welcome to file new RFCs with us!
```

Comment:
Choosing between "because of" and "for", I feel like "because of" is clearer here, given "for" has so many different meanings under different contexts. WDYT? @Vissidarte-Herman :-)

Reply:
Hahaha. Finally, I have a chance to prove you are misled!!!

"We still lack a concrete design or implementation roadmap for **these projects**.". The "for" here does not suggest a causal relationship.

Reply:
Oops. I originally suggested "These are the projects that we would like to work on but still lack a concrete design or implementation roadmap for because of their complexity."

Reply:
I see. Thanks for the clarification. Does it make sense to add a "," between "for" and "because of" then, for better clarity?

Reply:
> "These are the projects that we would like to work on but still lack a decent design or implementation roadmap for (because of their complexity)."
> This is the best from my end. WDYT?

Sounds great!